src/components/NewCommentForm.tsx

```diff
@@ -1,6 +1,101 @@
-import React from 'react';
+import React, {
+  useState,
+  KeyboardEvent,
+  MouseEvent,
+  useContext,
+  useCallback,
+} from 'react';
+import classNames from 'classnames';
+
+import { Post } from '../types/Post';
+
+// api
+
+import { addComments } from '../api/api';
+import { Context } from './Context';
+
+type Props = {
+  selectedPost: Post
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  selectedPost,
+}) => {
+  const {
+    commentList,
+    setCommentList,
+    setCommentListError,
+  } = useContext(Context);
+
+  const [inputName, setInputName] = useState('');
+  const [inputEmail, setInputEmail] = useState('');
+  const [inputText, setInputText] = useState('');
+
+  // Dangers
+
+  const [inputNameDagers, setInputNameDagers] = useState(false);
+  const [inputEmailDagers, setInputEmailDagers] = useState(false);
+  const [inputTextDagers, setInputTextDagers] = useState(false);
+
+  // functions
+
+  const clearInputs = useCallback(() => {
+    setInputName('');
+    setInputEmail('');
+    setInputText('');
+  }, []);
```

Comment:
```suggestion
  const clearInputs = () => {
    setInputName('');
    setInputEmail('');
    setInputText('');
  };
```
In my opinion useCallback is redundant here